ARCs/arc-0004.md

```diff
@@ -516,6 +516,7 @@ any leading zeros, _unless_ `N` is zero, in which case only a single 0 character
 * `address`: Used to represent a 32-byte Algorand address. This is equivalent to `byte[32]`.
 * `<type>[]`: A variable-length array. `type` can be any other type.
 * `string`: A variable-length byte array (`byte[]`) assumed to contain UTF-8 encoded content.
+* `bigint`: A variable-length byte array (`byte[]`) assumed to contain a variable-width unsigned integer.
```

Comment:
Perhaps this should be named `ubigint` to indicate that it's unsigned?

At some point in the future we may want to support signed ints, so it's good practice to avoid ambiguity.

Reply:
Yeah good point. Fixed in 89f3d51